src/stream/src/executor/mview/materialize.rs

```diff
@@ -111,9 +116,16 @@ impl<S: StateStore> MaterializeExecutor<S> {
                 pk_indices: arrange_columns,
                 identity: format!("MaterializeExecutor {:X}", executor_id),
             },
+            do_sanity_check: false,
         }
     }
 
+    /// Enable sanity check on this storage table.
+    /// Todo: add cache and do sanity check
+    pub fn enable_sanity_check(&mut self) {
+        self.do_sanity_check = true;
+    }
+
```

Comment:
I do not think we need this function and that behavior should be determined when `MaterializeExecutor` is constructed and we will not change it.